Workflows/.OwlBot.yaml

```diff
@@ -0,0 +1,5 @@
+deep-copy-regex:
+    - source: /google/cloud/workflows/(v1beta)/.*-php/(.*)
+      dest: /owl-bot-staging/Workflows/$1/$2
+    - source: /google/cloud/workflows/executions/(v1beta)/.*-php/(.*)
+      dest: /owl-bot-staging/Workflows/$1/Executions/$1/$2
```

Comment:
Are you sure about this destination? It implies that the version exists in the path twice, and when I look at the existing code, that is not the case

Reply:
Yes. I'm sure. It's just for gracefully handling the admin code in the common synthpy code.

Reply:
I can also remove the second version part too. I'll do that.

Reply:
@bshaffer Ok, I removed the second version part. Now it looks cleaner, thanks!